catalog/config.js.example

```diff
@@ -7,4 +7,5 @@ window.QUILT_CATALOG_CONFIG = {
   "passwordAuth": "ENABLED",
   "ssoAuth": "SIGN_IN_ONLY",
   "ssoProviders": "google",
+  "stackVersion": "1.55.0"
```

Comment:
```suggestion
  "stackVersion": "local-dev"
```
let's set this to smth easily distinguishable for local instances by default to avoid confusing sentry and mixpanel